OrbitService/ProcessServiceImpl.cpp

```diff
@@ -89,3 +91,60 @@ Status ProcessServiceImpl::GetProcessMemory(
             request->address(), request->pid()));
   }
 }
+
+Status ProcessServiceImpl::GetSymbols(ServerContext*,
+                                      const GetSymbolsRequest* request,
+                                      GetSymbolsResponse* response) {
+  std::shared_ptr<Module> module = nullptr;
+  {
+    absl::MutexLock lock(&mutex_);
+
+    const auto& process = process_list_.GetProcess(request->process_id());
+    if (!process) {
+      std::string error_message =
+          absl::StrFormat("The process with pid=%d was not found on remote",
+                          request->process_id());
+      ERROR("%s", error_message);
+      return Status(StatusCode::NOT_FOUND, error_message);
+    }
+
+    process->ListModules();
+    module = process->GetModuleFromPath(request->module_path());
+    if (!module) {
+      std::string error_message =
+          absl::StrFormat("The module with path \"%s\" was not found on remote",
+                          request->module_path());
+      ERROR("%s", error_message);
+      return Status(StatusCode::NOT_FOUND, error_message);
+    }
+  }
+
```

Comment:
I do not think you need to look up a Module here. You have full path and this is the only thing you want, isn't it?

If you want to preserve the interface to SymbolHelper in this PR - just create a module here - use it to load symbols and destroy it afterwards.

Reply:
This does make a lot of sense indeed. I used a Module there to not change SymbolHelper, but I wrote a comment and the change of SymbolHelper should be a quick other PR